projects/plugins/jetpack/modules/custom-css/custom-css.php

```diff
@@ -427,31 +462,33 @@ static function get_current_revision() {
 	 * Save new revision of CSS
 	 * Checks to see if content was modified before really saving
 	 *
-	 * @param string $css
-	 * @param bool $is_preview
+	 * @param string $css - the CSS.
+	 * @param bool   $is_preview - if we're in preview mode.
+	 * @param string $preprocessor - what preprocessor we're using.
 	 * @return bool|int If nothing was saved, returns false. If a post
 	 *                  or revision was saved, returns the post ID.
 	 */
-	static function save_revision( $css, $is_preview = false, $preprocessor = '' ) {
-		$safecss_post = Jetpack_Custom_CSS::get_post();
+	public static function save_revision( $css, $is_preview = false, $preprocessor = '' ) {
+		$safecss_post = self::get_post();
 
-		$compressed_css = Jetpack_Custom_CSS::minify( $css, $preprocessor );
+		$compressed_css = self::minify( $css, $preprocessor );
 
 		// If null, there was no original safecss record, so create one
-		if ( null == $safecss_post ) {
-			if ( ! $css )
+		if ( null == $safecss_post ) { // phpcs:ignore Universal.Operators.StrictComparisons.LooseEqual
```

Comment:
Other than `"0"`, this is equivalent. And `self::get_post()` should only be returning null or a possibly-empty array.
```suggestion
		if ( ! $safecss_post ) {
```